skipper.go

```diff
@@ -891,6 +891,39 @@ type Options struct {
 	LuaSources []string
 }
 
+func (o *Options) KubernetesDataClientOptions() kubernetes.Options {
```

Comment:
Just one question here, is it better to have this as Options function or to send options reference?